Representor/HTTP/HTTPRequest.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  HTTPRequest.swift
+//  Representor
+//
+//  Created by Kyle Fuller on 30/01/2015.
+//  Copyright (c) 2015 Apiary. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import URITemplate
+
+extension HTTPTransition {
+  var suggestedParameterEncoding:ParameterEncoding {
+    let encodingMapping = [
+      "application/json": ParameterEncoding.JSON,
```

Comment:
I just wonder, would it make a sense for this to work with JSON based media types in a future `xxx+json`?